### PR TITLE
header changes, new states for graphs, query in dashboard, and graph test data, graphs refactored

### PR DESCRIPTION
**Pull request by jojecameron**

### CODE REVIEW

File: src/components/Graphs/oldGraphs/LineChart.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ const LineChart: React.FC = () => {
   return (
     <div className="h-78  w-78 my-2 rounded-lg border border-gray-900  bg-red-100 shadow">
       <iframe
-        src="http://localhost:3000/d-solo/a2dc4fc5-613f-4527-b267-a01f274f4612/dvdrental-dashboard?orgId=1&from=1687300967135&to=1687322567135&panelId=4"
+        src="http://localhost:3000/d-solo/bce239a6-b14d-4359-a4f4-7bca1820ab2d/queryiq-for-database-performance-metrics-for-dvd-rental?orgId=1&from=1687776408132&to=1687798008132&panelId=3"
```
Comment (connoro7) [inline src/components/Graphs/oldGraphs/LineChart.tsx:8]: hardcoded

File: src/components/Graphs/oldGraphs/LineChart2.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ const LineChart2: React.FC = () => {
   return (
     <div className="h-78 w-78 my-2 rounded-lg border border-gray-900  bg-red-100 shadow">
       <iframe
-        src="http://localhost:3000/d-solo/a2dc4fc5-613f-4527-b267-a01f274f4612/dvdrental-dashboard?orgId=1&from=1687300979141&to=1687322579141&panelId=3"
+        src="http://localhost:3000/d-solo/bce239a6-b14d-4359-a4f4-7bca1820ab2d/queryiq-for-database-performance-metrics-for-dvd-rental?orgId=1&from=1687776441081&to=1687798041081&panelId=4"
```
Comment (connoro7) [inline src/components/Graphs/oldGraphs/LineChart2.tsx:8]: hardcoded

File: src/components/Graphs/oldGraphs/LineChart3.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ const LineChart3: React.FC = () => {
   return (
     <div className="h-78 w-78 my-2 rounded-lg border border-gray-900  bg-red-100 shadow">
       <iframe
-        src="http://localhost:3000/d-solo/a2dc4fc5-613f-4527-b267-a01f274f4612/dvdrental-dashboard?orgId=1&from=1687300995261&to=1687322595261&panelId=1"
+        src="http://localhost:3000/d-solo/bce239a6-b14d-4359-a4f4-7bca1820ab2d/queryiq-for-database-performance-metrics-for-dvd-rental?orgId=1&from=1687776468593&to=1687798068593&panelId=1"
```
Comment (connoro7) [inline src/components/Graphs/oldGraphs/LineChart3.tsx:8]: hardcoded

File: src/components/Graphs/oldGraphs/LineChart4.tsx [hunk 4]
```diff
@@ -1,11 +1,11 @@
 import React from "react";
 
 //Pie chart for Highest Queries by Memory Usage
-const PieChart2: React.FC = () => {
+const LineChart4: React.FC = () => {
   return (
     <div className="h-78 w-78 my-2 rounded-lg border border-gray-900  bg-red-100 shadow">
       <iframe
-        src="http://localhost:3000/d-solo/a2dc4fc5-613f-4527-b267-a01f274f4612/dvdrental-dashboard?orgId=1&from=1687301107627&to=1687322707627&panelId=2"
+        src="http://localhost:3000/d-solo/bce239a6-b14d-4359-a4f4-7bca1820ab2d/queryiq-for-database-performance-metrics-for-dvd-rental?orgId=1&from=1687776520531&to=1687798120531&panelId=9"
```
Comment (connoro7) [inline src/components/Graphs/oldGraphs/LineChart4.tsx:8]: hardcoded

File: src/components/Graphs/oldGraphs/LineChart5.tsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+//Pie chart for Highest Queries by Memory Usage
+const LineChart5: React.FC = () => {
+  return (
+    <div className="h-78 w-78 my-2 rounded-lg border border-gray-900  bg-red-100 shadow">
+      <iframe
+        src="http://localhost:3000/d-solo/bce239a6-b14d-4359-a4f4-7bca1820ab2d/queryiq-for-database-performance-metrics-for-dvd-rental?orgId=1&from=1687776643111&to=1687798243111&panelId=8"
```
Comment (connoro7) [inline src/components/Graphs/oldGraphs/LineChart5.tsx:8]: hardcoded

File: src/components/Graphs/oldGraphs/PieChart.tsx [hunk 6]
```diff
@@ -5,7 +5,7 @@ const PieChart: React.FC = () => {
   return (
     <div className="h-78 w-78 my-2 rounded-lg border border-gray-900  bg-red-100 shadow">
       <iframe
-        src="http://localhost:3000/d-solo/a2dc4fc5-613f-4527-b267-a01f274f4612/dvdrental-dashboard?orgId=1&from=1687300944489&to=1687322544489&panelId=5"
+        src="http://localhost:3000/d-solo/bce239a6-b14d-4359-a4f4-7bca1820ab2d/queryiq-for-database-performance-metrics-for-dvd-rental?orgId=1&from=1687776561866&to=1687798161866&panelId=2"
```
Comment (connoro7) [inline src/components/Graphs/oldGraphs/PieChart.tsx:8]: hardcoded

File: src/components/InputQuery.tsx [hunk 7]
```diff
@@ -28,14 +29,22 @@ const InputQuery: React.FC<InputQueryProps> = ({
...
     await asyncLoadingSim();
-    setQueryLog((prevQueryLog) => [
-      ...prevQueryLog,
-      { query: query, data: {}, name: "" },
+    const newQuery = { query: query, data: [], name: '' };
+    setQueryLog((prevQueryLog) => [...prevQueryLog, newQuery]);
+    setQuery('');
+    setActiveQuery(newQuery);
+    // hardcoded query results from: SELECT SUM(Payment.amount) AS sum_amt FROM payment
+    setQueryGraphs([
```
Comment (connoro7) [inline src/components/InputQuery.tsx:42]: hardcoded

**Review by khailetran — APPROVED** [review]

**Pull request merged**